index.html

```diff
@@ -12,6 +12,6 @@
   // var BUCKET_URL = 'https://BUCKET.s3.amazonaws.com';
   // var S3B_ROOT_DIR = 'SUBDIR_L1/SUBDIR_L2/';
 </script>
-<script src="http://rgrp.github.io/s3-bucket-listing/list.js"></script>
+<script src="http://rawgit.com/rgrp/s3-bucket-listing/gh-pages/list.js"></script>
```

Comment:
I just changed default url. So later, if anyone need https - it can be set as https (jquery link as well).
Your link just redirects https to http so it does not have any sense in setting it as https, it will always load http.